client.go

```diff
@@ -133,6 +133,10 @@ type ClientOptions struct {
 	TracesSampleRate float64
 	// Used to customize the sampling of traces, overrides TracesSampleRate.
 	TracesSampler TracesSampler
+	// Control with URLs trace propagation should be enabled. Does not support regex patterns.
+	TracePropagationTargets []string
+	// When set to true, the SDK will start a span for outgoing HTTP OPTIONS requests.
+	TraceOptionsRequests bool
```

Comment:
Why did you add this option?

Reply:
Saw it on the dev docs. Should I not implement that?

Reply:
Link?

Reply:
Here: https://develop.sentry.dev/sdk/telemetry/traces/#traceoptionsrequests

Reply:
> When set to true **transactions should be created** for HTTP OPTIONS requests.

This option does not apply to outgoing HTTP requests.

Reply:
Okay. Will update that later.